pipeline.config

```diff
@@ -14,14 +14,20 @@ CI:
   REPORT_PATH: "lcov.info"
   BUILDCACHE: "false"
   REPLACEARGS: "true"
-  VAULT_KEYS_TO_INJECT: "IBM_GITHUB_PUBLIC_USERNAME,IBM_GITHUB_PUBLIC_TOKEN"
+  # FHIR_CLUSTER_NAMESPACE is set in keyProtect to override the default behavior that requires the fhir server
```

Comment:
Something about the name`FHIR_CLUSTER_NAMESPACE` isn't  meshing well with my brain. Would `FHIR_SERVER_NAMESPACE` be an alternative here?

Reply:
The reason I chose FHIR_CLUSTER_NAMESPACE is that the default env var provided by the toolchain for the base namespace we are deploying to is CLUSTER_NAMESPACE, so I added FHIR to it to maintain that continuity. To me it seemed to mean the namespace in the cluster, or the fhir namespace within the cluster.

Reply:
Works for me.

I think my brain was reading `[FHIR Cluster] namespace` (as if an entirely separate cluster existed for FHIR and this was a special namespace there). I should be able to retrain my brain to see it as  `FHIR [cluster namespace]`.  😄